downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/CellularComponentAnnotationBuilder.java

```diff
@@ -59,17 +49,17 @@ private static void processProteins(Set<GKInstance> proteinInstances, GKInstance
 
     /**
      * Proteins reaching this part of the script are deemed valid for GO annotation. Retrieves the Cellular Compartment accession associated with this instance and then calls the GOA line generator.
-     * @param proteinInst -- Individual protein instance from the retrieved proteins.
-     * @param referenceEntityInst -- Reference Entity instance of the protein.
-     * @param reactionInst -- Parent reaction that the protein is assocaited with.
-     * @param taxonIdentifier -- CrossReference ID of protein's species
-     * @throws Exception
+     * @param proteinInst -- GKInstance, Individual protein instance from the retrieved proteins.
+     * @param referenceEntityInst -- GKInstance, Reference Entity instance of the protein.
```

Comment:
I'm wondering if this parameter is needed since it can be obtained from the proteinInst parameter?  Shorter parameter lists where possible are usually desirable.